workshops/networking/2.1-towerinstall/README.md

```diff
@@ -0,0 +1,85 @@
+# Exercise 2.1 - Installing Ansible Tower
+
+In this exercise, we are going to get Ansible Tower installed on your control node
+
+## Step 1: ssh into your control node
+
+```bash
+ssh <username>@<IP of control node>
+```
+
+## Step 2: Change directories to /tmp
+
+```bash
+cd /tmp
+```
+
+## Step 3: Download the latest Ansible Tower package
+
+```bash
+curl -O https://releases.ansible.com/ansible-tower/setup-bundle/ansible-tower-setup-bundle-latest.el7.tar.gz
```

Comment:
This is inconsistent with the slides. @dfederlein setup a shortcut url to make this easy to type.

Reply:
@tima is this the bit.ly link here: wget https://bit.ly/ansibletower ?  Usually short url shorteners by third party companies are a bad idea... there was a huge case of ShortURL being compromised and everyone getting rick rolled.  Couldn't we just get a short URL from ansible.com like ansible.com/gettowernow or something?

Reply:
@tima fixed with commit: https://github.com/network-automation/lightbulb/commit/cea7f6d8eda31038019c38fe3f9d524328fb14f0